test.sh

```diff
@@ -1,2 +1,11 @@
 #!/bin/bash
-exec python -m samplerunner test
+echo "#############################"
+echo "# DOWNLOADING DOCKER IMAGES #"
+echo "#############################"
+python -um samplerunner download
```

Comment:
I think I want to get rid of this file altogether and rewrite the travis file to leverage the two commands in this file in different build phases.

Reply:
Agreed